build.gradle

```diff
@@ -125,7 +125,6 @@ dependencies {
   implementation group: 'commons-beanutils', name: 'commons-beanutils', version: versions.commonsBeanUtils
 
   implementation group: 'uk.gov.service.notify', name: 'notifications-java-client', version: versions.hmctsNotify
-  implementation group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: versions.reformHealth
```

Comment:
As per note in https://github.com/hmcts/chart-java/blob/master/README.md#chart-java default sprint probes introduced in Spring Boot 2.3.x should be used